### DISCUSSION TITLE
Streak not working

### DISCUSSION THREAD

**Discussion by krishna1584** [community/community#136012] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My github streak is not working. I have been making commits even though its not updating. 2 3 days back I back dated a  commit is it due to this? If so please tell me how to resolve the issue.

**Comment by Nuwan-Srimal** [comment, marked as answer]
Yes, backdating commits can affect your GitHub streak. GitHub tracks your contribution activity based on the dates of your commits. If you change the date of a commit to a date outside your current streak period, it can break the streak.
